src/lib/util/parseStringBank.ts

```diff
@@ -80,7 +87,7 @@ export function parseBank({ inputBank, inputStr, flags = {} }: ParseBankOptions)
 		for (const [item, quantity] of strItems) {
 			_bank.add(
 				item.id,
-				!quantity ? inputBank.amount(item.id) : Math.max(1, Math.min(quantity, inputBank.amount(item.id)))
+				!quantity ? inputBank.amount(item.id) : Math.max(0, Math.min(quantity, inputBank.amount(item.id)))
```

Comment:
This fixes the bug where `+b 1 Twisted bow` will show 1 twisted bow even if you don't have any.